isaric/parsers/ncov-phosp.toml

```diff
@@ -0,0 +1,803 @@
+  date = { field = "crf1a_date_adm" }
+  phase = "admission"
+  is_present = { field = "crf1a_join_pain", ref = "Y/N" }
+
+[[observation]]
+  name = "joint_pain"
+  date = { field = "patient_sq_date" }
+  phase = "followup"
+  is_present = { field = "joint_pain_or_swelling", ref = "Y/N" }
+
+[[observation]]
+  name = "conjunctivitis"
+  phase = "admission"
+  date = { field = "crf1a_date_adm" }
+  is_present = { field = "crf1a_conjunctivitis", ref = "Y/N" }
+
+[[observation]]
+  name = "loss_of_smell"
+  date = { field = "crf1a_date_adm" }
+  phase = "admission"
+  is_present = { field = "crf1a_loss_smell", ref = "Y/N" }
+
+[[observation]]
+  name = "loss_of_taste"
+  date = { field = "crf1a_date_adm" }
+  phase = "admission"
+  is_present = { field = "crf1a_loss_taste", ref = "Y/N" }
+
+[[observation]]
+  name = "loss_of_taste"
+  date = { field = "patient_sq_date" }
+  phase = "followup"
+  is_present = { field = "loss_of_taste", ref = "Y/N" }
+
+[[observation]]
+  name = "loss_of_smell_or_taste"
+  date = { field = "crf1a_date_adm" }
+  phase = "admission"
+
+  [observation.is_present]
+    combinedType = "any"
+    fields = [
+      { field = "crf1a_loss_taste", ref = "Y/N" },
+      { field = "crf1a_loss_smell", ref = "Y/N" },
+    ]
+
+[[observation]]
+  name = "loss_of_smell_or_taste"
+  date = { field = "patient_sq_date" }
+  phase = "followup"
+  is_present = { field = "loss_of_smell_or_taste" }
+
+[[observation]]
+  name = "lower_chest_wall_indrawing"
+  date = { field = "crf1a_date_adm" }
+  phase = "admission"
+  is_present = { field = "crf1a_chest_indrawing", ref = "Y/N" }
+
+[[observation]]
+  name = "lymphadenopathy"
+  date = { field = "crf1a_date_adm" }
+  phase = "admission"
+  is_present = { field = "crf1a_lymphadenopathy", ref = "Y/N" }
+
+[[observation]]
+  name = "muscle_aches"
+  date = { field = "crf1a_date_adm" }
+  phase = "admission"
+  is_present = { field = "crf1a_muscle_aches", ref = "Y/N" }
+
+[[observation]]
+  name = "muscle_aches"
+  date = { field = "patient_sq_date" }
+  phase = "followup"
+  is_present = { field = "aching_in_your_muscles_pai", ref = "Y/N" }
+  
+[[observation]]
+  name = "runny_nose"
+  date = { field = "crf1a_date_adm" }
+  phase = "admission"
+  is_present = { field = "crf1a_runny_nose", ref = "Y/N" }
+
+[[observation]]
+  name = "seizures"
+  date = { field = "crf1a_date_adm" }
+  phase = "admission"
+  is_present = { field = "crf1a_seizures", ref = "Y/N" }
+
+[[observation]]
+  name = "seizures"
+  date = { field = "patient_sq_date" }
+  phase = "followup"
+  is_present = { field = "seizures", ref = "Y/N" }
+
+[[observation]]
+  name = "severe_dehydration"
+  date = { field = "crf1a_date_adm" }
+  phase = "admission"
+  is_present = { field = "crf1a_sev_dehydr", ref = "Y/N" }
+
+[[observation]]
+  name = "oxygen_o2hb"
+  date = { field = "crf1a_date_adm" }
+  phase = "admission"
+  value = { field = "crf1a_haemoglobin" }
+  if = { crf1a_haemoglobin = { "!=" = "NA" } }
```

Comment:
@ekamau should a context field be added perhaps, to indicate if the measured value is less than or greater than the value indicated in crf1a_haemoglobin? This is provided in crf1a_haemo_less_great for 24 records

Reply:
added context field here .. but I am not sure if I did it correctly. 
The values are in the 'crf1a_haemoglobin' field are text .. I wasn't sure how to map them in the parser.

@pipliggins - could you please help me check if this has been done correctly?